.github/workflows/coverage.yaml

```diff
@@ -33,6 +33,7 @@ jobs:
           MAVEN_OPTS: -Dorg.slf4j.simpleLogger.defaultLogLevel=warn -Dhttp.keepAlive=false -Dmaven.wagon.http.pool=false -Dmaven.wagon.httpconnectionManager.ttlSeconds=120
         run: mvn clean test  --batch-mode
       - name: Test Report
+        continue-on-error: true
```

Comment:
This will go away once https://github.com/DataBiosphere/consent/pull/2687 is completed.